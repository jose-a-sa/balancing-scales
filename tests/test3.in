#ScaleName,Left,Right
A1,5,B1
B1,6,B2
B2,B3,4
B3,8,B4
B4,1,B5
B5,2,20