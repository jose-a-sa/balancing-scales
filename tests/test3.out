A1,666,0
B1,329,0
B2,0,163
B3,75,0
B4,40,0
B5,18,0
